renderdoc/driver/vulkan/wrappers/vk_cmd_funcs.cpp

```diff
@@ -25,6 +25,7 @@
 #include "../vk_core.h"
 #include "../vk_debug.h"
 #include "core/settings.h"
+#include "driver/vulkan/vk_common.h"
```

Comment:
For "local" includes in the driver like this please do `../vk_common.h` as in the other includes.